src/components/FileUploadModal/style.css

```
@@ -33,3 +33,11 @@
 .upload-modal :global(.ant-tabs .ant-tabs-tab-active) {
     font-weight: 400;
 }
+
+.file-dragged:global(.ant-modal-wrap) {
+    pointer-events: none;
+}
+
+.file-dragged :global(.ant-modal-content) {
+    pointer-events: none;
+}
```

optional: could be a single selector joined with a comma:


```suggestion
.file-dragged:global(.ant-modal-wrap), .file-dragged :global(.ant-modal-content) {
    pointer-events: none;
}
```